src/argus/incident/models.py

```diff
@@ -338,6 +338,7 @@ class Type(models.TextChoices):
     incident = models.ForeignKey(to=Incident, on_delete=models.PROTECT, related_name="events")
     actor = models.ForeignKey(to=User, on_delete=models.PROTECT, related_name="caused_events")
     timestamp = models.DateTimeField()
+    received = models.DateTimeField(default=timezone.now())
```

Comment:
This should probably be changed to the following, to avoid the constant timestamp in the migration 😅 
```suggestion
    received = models.DateTimeField(auto_now_add=True)
```

Reply:
Naw, removing the "()" is sufficient.

I don't like using "auto_now*" anything because fun things might happen when you dump/load fixtures, though "auto_now" is much worse in that regard.

Reply:
I've no experience with using `auto_now`, but I do agree on dropping the `()` from `now()` to achieve the intended result.

Reply:
Fixed